qiita_pet/handlers/study_handlers/listing_handlers.py

```diff
@@ -72,8 +73,12 @@ def _build_study_info(user, search_type, study_proc=None, proc_samples=None):
 
     # get list of studies for table
     if search_type == 'user':
-        user_study_set = user.user_studies.union(user.shared_studies)
-        study_set = user_study_set - Study.get_by_status('public')
+        if user.level != 'admin':
+            user_study_set = user.user_studies.union(user.shared_studies)
+            study_set = user_study_set - Study.get_by_status('public')
+        else:
+            portal_set = Portal(qiita_config.portal).get_studies()
```

Comment:
Why is this line needed? `Study.get_by_status` should already return the studies that are only visible in the current portal. If not, there is a bug in that function so I would prefer to change that function rather than patch it here.